config/puma.rb

```diff
@@ -20,10 +22,10 @@
 end
 
 # Specifies the `environment` that Puma will run in.
-environment ENV.fetch("RAILS_ENV") { "development" }
+environment ENV.fetch("RAILS_ENV", "development")
```

Comment:
❤️

Reply:
Interesting, not sure why `rubocop -A` decided to fix that now 🤔